src/counter_analysis_toolkit/src/Makefile

```diff
@@ -92,15 +92,15 @@ flops: flops.c flops.h cat_arch.h
 	$(CC) $(CFLAGS) $(FLOP) $(OPT1) $(INCFLAGS) -c flops.c -o flops.o
 
 icache.o: icache.c icache.h
-	bash gen_seq_dlopen.sh
+	bash gen_seq_dlopen.sh ${PWD}
 	$(CC) $(CFLAGS) $(OPT0) $(INCFLAGS) -c icache.c -o icache.o
 
-icache_seq_kernel_0.o: icache_seq.c icache_seq.h
+icache_seq_kernel.o: icache_seq.c icache_seq.h icache_seq_kernel.c
 	$(CC) $(CFLAGS) $(OPT0) $(INCFLAGS) -c icache_seq.c -o icache_seq.o
-	$(CC) $(CFLAGS) $(OPT0) $(INCFLAGS) -fPIC -c icache_seq_kernel.c -o icache_seq_kernel_0.o
-	$(CC) $(CFLAGS) $(OPT0) -shared -o icache_seq_kernel_0.so icache_seq_kernel_0.o
-	bash replicate.sh
-	rm icache_seq_kernel_0.o
+	$(CC) $(CFLAGS) $(OPT0) $(INCFLAGS) -fPIC -c icache_seq_kernel.c -o icache_seq_kernel.o
+	$(CC) $(CFLAGS) $(OPT0) -shared -o icache_seq_kernel_0.so icache_seq_kernel.o
```

Comment:
You removed 0 from the `icache_seq_kernel.o` filenames, do you need to remove it from `icache_seq_kernel_0.so`?